VideoLocker/src/main/java/org/edx/mobile/user/Account.java

```diff
@@ -0,0 +1,201 @@
+package org.edx.mobile.user;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Account {
+    @SerializedName("username")
+    @NonNull
```

Comment:
Silent-failures (like using a blank string when we expected to get a string from the API) are a worse problem then a crashing app, because we won't know that there is anything wrong.

What if we configured Gson to fail if `@NonNull` field has a `null` value? Then it can be treated at runtime like any other deserialization error.